### ISSUE TITLE
Text Assembly across multiple lines

### ISSUE THREAD

**Issue by jelliottp** [SAP/code-pal-for-abap#597]
**Check Name**
Text Assembly

**Actual Behavior**
Text assembly alerts when `|` is properly used in combination with `&&`, because text is on multiple lines due to line length limits in place.

**Expected Behavior**
If text assembly is split across multiple lines while using`|`, I would not want a finding reported. I could submit a PR for this if there is agreement.

**Comment by bjoern-jueliger-sap** [comment]
I think you mean that the check yields false positives on code like

```abap
data(str) = |this is a very long string with some { placeholder }| &&
    |and this is the rest of the string with another { placeholder }|.
```
right? If so, yeah, that's a bug and if you want to fix it you're very welcome :)

**Comment by jelliottp** [comment]
> I think you mean that the check yields false positives on code like
> 
> ```abap
> data(str) = |this is a very long string with some { placeholder }| &&
>     |and this is the rest of the string with another { placeholder }|.
> ```
> 
> right? If so, yeah, that's a bug and if you want to fix it you're very welcome :)

Correct. I'll look into it further then.

**Comment by bjoern-jueliger-sap** [comment]
Fixed with #598, thanks!
